titanium-address-input.html

```diff
@@ -320,6 +339,12 @@
 					this.autocomplete = new google.maps.places.Autocomplete(input, {
 						types: ['address']
 					});
+
+					/* Allows addresses to be limited to a particular country or list of countries */
+					this.autocomplete.setComponentRestrictions({
```

Comment:
Could we do this on an observer on limitToCountries so that you can change the list of countries on the fly?